packages/database/src/migrations/20240918030717-AddPacMOSSIentities-modifies-schema.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+import { TupaiaDatabase } from '../TupaiaDatabase';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function () {
+  const db = new TupaiaDatabase();
+  await db.executeSql(`ALTER TYPE public.entity_type ADD VALUE IF NOT EXISTS 'pacmossi_district';`);
+  await db.executeSql(`ALTER TYPE public.entity_type ADD VALUE IF NOT EXISTS 'pacmossi_village';`);
+  await db.executeSql(
+    `ALTER TYPE public.entity_type ADD VALUE IF NOT EXISTS 'pacmossi_spraying_site';`,
+  );
+  return db.closeConnections();
```

Comment:
```suggestion
exports.up = async function (db) { 
  return db.executeSql(`ALTER TYPE public.entity_type ADD VALUE IF NOT EXISTS 'pacmossi_district';
  ALTER TYPE public.entity_type ADD VALUE IF NOT EXISTS 'pacmossi_village';
  ALTER TYPE public.entity_type ADD VALUE IF NOT EXISTS 'pacmossi_spraying_site';`,
  );
```

We don't usually need to add or close a db connection here, as the `db` param is available to us. We can also add all the sql expressions in one go, instead of separately.